docs/getting-started.md

```diff
@@ -22,46 +22,52 @@ with soci-snapshotter.
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Prerequisites
+## Dependencies
 
-- **[go](https://go.dev/doc/install) >= 1.18** - to confirm please check with `go version`.
-- **[containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) >= 1.4** - to confirm that you have containerd working please check with
-`sudo ctr version`.
-- **[ctr](https://github.com/containerd/containerd/blob/main/docs/getting-started.md)/[nerdctl](https://github.com/containerd/nerdctl#install)** - you need one of the containerd clients to interact with containerd/registry.
-- **fuse** - used for mounting without root access (`sudo yum install fuse`).
-- **zlib** - used for decompression and ztoc creation (`sudo yum install zlib-devel zlib-static`).
-- **gcc** - used for compiling SOCI's c code, gzip's zinfo implementation (`sudo yum install gcc`).
+soci-snapshotter has the following runtime dependencies. Please follow the links or commands
+to install them on your machine:
+
+> We only mention the direct dependencies of the project. Some dependencies may
+> have their own dependencies (e.g., containerd depends on runc/cni). Please refer
+> to their doc for a complete installation guide (mainly containerd).
+
+- **[containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) >= 1.4** -
+required to run soci-snapshotter; to confirm please check with `sudo ctr version`.
```

Comment:
Sorry I should of mentioned this in my previous comment can we also update this to sudo containerd --version

Reply:
updated the `install.md` one to `sudo containerd --version`. I think it's fine to keep the `sudo ctr version` in getting-started since `ctr` is also a dep for this doc, it doesn't hurt to check both client/server versions. WDYT?

Reply:
I guess my only concern would be consistency, but I don't think its that big of deal.